src/winforms/toga_winforms/widgets/table.py

```diff
@@ -0,0 +1,37 @@
+from toga.interface import Table as TableInterface
+
+from ..libs import *
+from .base import WidgetMixin      
+
+class Table(TableInterface, WidgetMixin):
+    def __init__(self, headings, id=None, style=None):
+        super(Table, self).__init__(headings, id=id, style=style)
+        self._create()
+
+    def create(self):
+        self._container = self
+        self._impl = WinForms.DataGridView()
```

Comment:
I don't think this is the right widget; a TreeView is probably the better option here.